pylintrc

```diff
@@ -21,7 +21,11 @@ disable=duplicate-code,
         unused-argument,
         unused-variable,
         wrong-spelling-in-comment,
-        wrong-spelling-in-docstring
+        wrong-spelling-in-docstring,
+        # needed for explicit re-exports
+        useless-import-alias,
+        # false positives with generics
+        unsubscriptable-object
```

Comment:
Thank you for the proposal-- I'm a little hesitant to go full type-hints with trio-websocket.

There are benefits, but they doesn't come for free-- at the cost of more verbose code, more complexity, and more dependencies.

Things like the tedious re-exporting, or having to disable useful pylint checkers, are example downsides.  Also, it can be corrected, but it seems unnecessary to type hint variables and args that can be inferred by initialization (`max_message_size: int = MAX_MESSAGE_SIZE`, etc.).

May I ask if you're actively using trio-websocket in a project?  Also, did this exercise uncover any bugs in the implementation?

(This comment is overall for the PR, but I'll start the thread here.)

Reply:
> May I ask if you're actively using trio-websocket in a project?

Yep! Though I have just gotten by by including those stubs in a folder.

Project is open-source actually: https://github.com/A5rocks/bloom

>  Also, did this exercise uncover any bugs in the implementation?

As mentioned, I started with a base set of stubs so I didn't find any bugs in my implementation.

> There are benefits, but they doesn't come for free-- at the cost of more verbose code, more complexity, and more dependencies.

Yep, although I *should* be able to just extract the stubs out into `.pyi` files (I haven't played with mypy's `stubgen` and `stubtest` yet), if the downsides are too much.

Reply:
I don't know if external pyi files improves much.  `stubgen` would just provide a starting point, and would have to be manually edited and kept up to date with the source.  And it doesn't address some of the complexity issues (typing can get quite complex, such as nested types, async generators, etc.).